meta-balena-common/recipes-support/balena-info/balena-info.bb

```diff
@@ -23,18 +23,18 @@ do_build[noexec] = "1"
 
 do_install() {
     install -d ${D}${sbindir}/
-    install -m 0755 ${WORKDIR}/resin-info ${D}${sbindir}/
+    install -m 0755 ${WORKDIR}/balena-info ${D}${sbindir}/
 
     if ${@bb.utils.contains('DISTRO_FEATURES','systemd','true','false',d)}; then
         install -d ${D}${systemd_unitdir}/system/
         install -d ${D}${sysconfdir}/systemd/system/multi-user.target.wants/
-        install -m 0644 ${WORKDIR}/resin-info@.service ${D}${systemd_unitdir}/system/
+        install -m 0644 ${WORKDIR}/balena-info@.service ${D}${systemd_unitdir}/system/
 
         if ${@bb.utils.contains('DISTRO_FEATURES','development-image','true','false',d)}; then
```

Comment:
Should the 'development-image' here become 'osdev-image'?

Reply:
I must have missed that line in my first push, it's removed in the current one.